AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -26,6 +26,7 @@ quinhillyer.com,johnbiver.com,jasonmattera.com,ipatriot.com,hooligancharters.com
 ##.widget_blog_subscription
 ~curvegames.com,~news.un.org,~norwichzen.org.uk,~dereferer.me###mc_embed_signup
 !
+manutd.com##.custom-popup
```

Comment:
I don't see it on my end.

Reply:
Removed.
https://github.com/AdguardTeam/AdguardFilters/pull/104221/commits/a94484bd25b56c010b6893f5b1e0e485887ec522